src/dotnet/APIView/APIViewWeb/Pages/Assemblies/Review.cshtml

```diff
@@ -504,10 +506,90 @@
+                                            <i class="bi bi-plus-circle-fill created status-icon"></i>
+                                        }
+                                        else if (change.ChangeAction.ToString() == "Approved")
+                                        {
+                                            <i class="bi bi-check-circle-fill approved status-icon"></i>
+                                        }
+                                        else if (change.ChangeAction.ToString() == "ApprovalReverted")
+                                        {
+                                            <i class="bi bi-arrow-left-circle-fill approval-reverted status-icon"></i>
+                                        }
+                                        else if (change.ChangeAction.ToString() == "Deleted")
+                                        {
+                                            <i class="bi bi-backspace-fill deleted status-icon"></i>
+                                        }
+                                        else if (change.ChangeAction.ToString() == "UnDeleted")
+                                        {
+                                            <i class="bi bi-plus-circle-fill undeleted status-icon"></i>
+                                        }
+                                    </div>                                   
+                                    <div class="revision-events">
+                                        @if (change.ChangeAction.ToString() == "Created")
+                                        {
+                                            <span class="revision-info">Created</span><br />
+                                        }
+                                        else if (change.ChangeAction.ToString() == "Approved")
+                                        {
+                                            <span class="revision-info">Approved</span><br />                                                                                           
+                                        }
+                                        else if (change.ChangeAction.ToString() == "ApprovalReverted")
+                                        {
+                                            <span class="revision-info">Approval Reverted</span><br />                                            
+                                        }
+                                        else if (change.ChangeAction.ToString() == "Deleted")
+                                        {
+                                            <span class="revision-info">Deleted</span><br />
+                                        }
+                                        else if (change.ChangeAction.ToString() == "UnDeleted")
+                                        {
+                                            <span class="revision-info">UnDeleted</span><br />
+                                        }

```

Comment:
I would like this and lines from 519 - 535 optimized. It's same check. You can use a local variable to set the value for both  class name and span value using just one if check. Actually second one does not even require if else check for all statuses.